pkgs/development/node-packages/default.nix

```diff
@@ -415,6 +415,10 @@ let
       buildInputs = [ pkgs.libusb1 ];
     };
 
+    uppy-companion = super."@uppy/companion".override {
+      name = "uppy-companion";
```

Comment:
This is cumbersome. It sort of makes sense, but no other package does it.

Reply:
I stand corrected. `bitwarden-cli` does it as well, and I wonder why not more packages that have `@` and `/` in the attribute name have it. LGTM then.

Reply:
https://github.com/NixOS/nixpkgs/blob/6748e4d66376675407eaaaa2858786028242deeb/pkgs/development/node-packages/default.nix#L134-L137